lib/bap_dwarf/dwarf_fbi.mli

```diff
@@ -16,10 +16,10 @@ type fn [@@deriving bin_io, compare, sexp]
     - .debug_info   [Section.Info]
     - .debug_str    [Section.Str]
 *)
-val create : string Dwarf_data.t -> t Or_error.t
+val create : bytes Dwarf_data.t -> t Or_error.t
```

Comment:
let's keep `string`s in the interface. Especially, since we're not going to mutate them, it is better to use the `string` type to denote this.